benchmarks/StiffODE/Bruss.jmd

```diff
@@ -400,7 +456,7 @@ setups = [
 names = ["KenCarp47 KLU MTK", "KenCarp47 GMRES MTK",
          "FBDF KLU MTK", "FBDF GMRES MTK",
          "Rodas5P GMRES MTK",
-         "CVODE MTK KLU", "CVODE iLU MTK GMRES"
```

Comment:
why this change?

Reply:
Because that setup doesn't reference problem 2